echo/server/run-worker-cpu.sh

```
@@ -1 +1 @@
-dramatiq --queues cpu --processes 1 --threads 2 dembrane.tasks
\ No newline at end of file
+dramatiq --queues cpu --processes 2 --threads 1 dembrane.tasks
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)